bytecomp/translcore.ml

```diff
@@ -1284,7 +1284,11 @@ and transl_record loc env fields repres opt_init_expr =
         match repres with
         | Record_regular -> Lconst(Const_block(0, cl))
         | Record_inlined tag -> Lconst(Const_block(tag, cl))
-        | Record_unboxed _ -> Lconst(match cl with [v] -> v | _ -> assert false)
+        | Record_unboxed _ -> Lconst(
```

Comment:
What's the point of "unboxing" an empty record type, since the representation ends up being the same as the regular representation?